### ISSUE TITLE
sls deploy issue 

### ISSUE THREAD

**Issue by stephn2027** [serverless/serverless#12613]
### Issue description

Cannot deploy sls. States that too may open files. Increased ulimit -n by 4096 still having issues

### Context

Serverless ϟ Support

This generates a report from your last Serverless Framework command (including any errors) to use for Github Issues, debugging w/ AI, or creating a support ticket w/ Serverless Inc.

Summary Report -----------------

# Service Overview
- Serverless Framework Version: 4.1.4
- Service Config File: serverless.yml
- Service Name: conversation-insights-api
- Service App: insights
- Service Runtime: nodejs18.x
- Service Stage: dev
- Service Region: ap-northeast-1
- Command: deploy 
- Error Code: CANNOT_READ_FILE

## Service Path
/Users/kor/React Projects/conversation-insights/serverless

## Error Message
Cannot read file node_modules/lodash/fp/hasIn.js due to: EMFILE: too many open files, open '/Users/kor/React Projects/conversation-insights/serverless/node_modules/lodash/fp/hasIn.js'

## Error Stacktrace
```
at file:///Users/kor/.serverless/releases/4.1.4/package/dist/sf-core.js:792:26637
at tryCatcher (file:///Users/kor/.serverless/releases/4.1.4/package/dist/sf-core.js:374:81933)
at Promise2._settlePromiseFromHandler (file:///Users/kor/.serverless/releases/4.1.4/package/dist/sf-core.js:552:10136)
at Promise2._settlePromise (file:///Users/kor/.serverless/releases/4.1.4/package/dist/sf-core.js:552:11455)
at Promise2._settlePromise0 (file:///Users/kor/.serverless/releases/4.1.4/package/dist/sf-core.js:552:12512)
at Promise2._settlePromises (file:///Users/kor/.serverless/releases/4.1.4/package/dist/sf-core.js:552:14108)
at _drainQueueStep (file:///Users/kor/.serverless/releases/4.1.4/package/dist/sf-core.js:381:583)
at _drainQueue (file:///Users/kor/.serverless/releases/4.1.4/package/dist/sf-core.js:381:479)
at Async._drainQueues (file:///Users/kor/.serverless/releases/4.1.4/package/dist/sf-core.js:381:716)
at Async.drainQueues [as _onImmediate] (file:///Users/kor/.serverless/releases/4.1.4/package/dist/sf-core.js:377:3278)
at process.processImmediate (node:internal/timers:476:21)
```

## Service Config
```
# "org" ensures this Service is used with the correct Serverless Framework Access Key.
org: innovativelearnings
# "app" enables Serverless Framework Dashboard features and sharing them with other Services.
app: insights
service: conversation-insights-api
useDotenv: true
provider:
  name: aws
  profile: ilang
  runtime: nodejs18.x
  logRetentionInDays: 7
  role: arn:aws:iam::105150981739:role/LambdaCloudWatchLogsRole
  environment:
    OPENAI_API_KEY: ${env:OPENAI_API_KEY}
    AIRTABLE_API_KEY: ${env:AIRTABLE_API_KEY}
    DEEPGRAM_API_KEY: ${env:DEEPGRAM_API_KEY}
    AWS_SDK_LOAD_CONFIG: 1
  region: ${env:AWS_REGION_ID}
  iamRoleStatements:
    - Effect: "Allow"
      Action:
        - "logs:CreateLogGroup"
        - "logs:CreateLogStream"
        - "logs:PutLogEvents"
      Resource: "arn:aws:logs:${env:AWS_REGION_ID}:*:*"
  httpApi:
    cors: true
functions:
  generateAudio:
    handler: generateAudio.handler
    events:
      - http:
          method: post
          path: generateaudio
  feedback:
    handler: feedback.handler
    timeout: 25
    events:
      - http:
          method: post
          path: feedback
  awsTranscribe:
    handler: awsTranscribe.handler
    events:
      - http:
          method: post
          path: awstranscribe
  deepTranscribe:
    handler: deepTranscribe.handler
    events:
      - http:
          method: post
          path: deeptranscribe
  transcribe:
    handler: transcribe.handler
    events:
      - http:
          method: post
          path: transcribe
  getDialogue:
    handler: getDialogue.handler
    events:
      - http:
          method: get
          path: dialogue
  romanize:
    handler: romanize.handler
    events:
      - http:
          method: post
          path: romanize
  analyze:
    handler: analyze.handler
    events:
      - http:
          method: post
          path: analyze
  offRails: 
    handler: offRails.handler
    events:
      - http:
          method: post
          path: offRails
  generateVideo:
    handler: generateVideo.handler
    events:
      - http:
          method: post
          path: generatevideo
  saveAudio:
    handler: saveAudio.handler
    events:
      - http:
          method: post
          path: saveaudio
plugins:
  - serverless-offline
  - serverless-dotenv-plugin
package:
  include:
    - bin/**
custom:
  serverless-offline:
    noTimeout: true
```

**Comment by garethmcc** [comment]
@stephn2027 the serverless-dotenv-plugin should no longer be required as those features were added to Serverless Framework natively and may assist in resolving that issue.

**Comment by skierkowski** [comment]
This issue was resolved as of 4.1.12

**Comment by stephn2027** [comment]
Thank you. will update to v4.1.12
